src/docs/product/accounts/quotas/manage-event-stream-guide.mdx

```diff
@@ -15,177 +15,52 @@ Sending all your application errors to Sentry ensures you'll be notified in real
+## 2. Event Usage Stats {#common-workflows-for-managing-your-event-stream}
 
-To learn more and see code samples, check out:
+Once you've completed the steps above, you can look at your events in aggregate in the "Usage Stats" tab of **Stats**. This information will help you answer some key questions like:
 
-- [Filtering](/platforms/javascript/configuration/filtering/)
-- [Enriching Events: Add Context](/platforms/javascript/enriching-events/context/)
+- What's the breakdowndown of my incoming events?
+- Which projects are consuming my quota?
+- Which issues are consuming my quota?
 
-#### GlobalHandlers
+The answers to these questions can help you figure out where you need to do further fine-tuning of your filters, limits, or SDK configuration.
 
-The integration attaches global handlers to capture uncaught exceptions (`onerror`) and unhandled rejections (`onunhandledrejection`). Both handlers are enabled by default, but can be disabled through configuration. Learn more in [GlobalHandlers Integration](/platforms/javascript/configuration/integrations/default/#globalhandlers).
+## 3. Quota Availability
 
-Check out additional configuration options with the [tryCatch](/platforms/javascript/configuration/integrations/default/#trycatch) and [ReportingObserver](/platforms/javascript/configuration/integrations/plugin/#reportingobserver) integrations.
+blah blah
 
-### Other SDKs
+## 4. Rate Limiting
 
-**Java** - Sentry's [Java SDK](/platforms/java/) provides integrations with common Java logging libraries. The configuration allows you to set a logging threshold determining the minimum level to capture a log message as breadcrumb. It also provides a setting to configure the minimum log level to capture an event.
+Rate limiting allows you to [limit the amount of events Sentry accepts](/product/accounts/quotas/#what-counts-toward-my-quota-table-view) per project for a defined time span. While this is quite useful for managing your monthly event quota, keep in mind that once a defined threshold is crossed, **subsequent events will be dropped**. Therefore, you shouldn't constantly be hitting your rate limit; rather, it should act as a ceiling intended to protect you from unexpected spikes.
 
-**PHP** - The `error_types` configuration option allows you to set the error types you want Sentry to monitor. Learn more in PHP [Basic Options](/platforms/php/configuration/options/#common-options).
+## 5. Delete & Discard Issues {#5-applying-workflows}
 
-**Ruby** - The `excluded_exceptions` configuration option allows you to set the exception types you wish to suppress. Learn more in the [Ruby configuration docs](/platforms/ruby/configuration/options/#optional-settings)
+Now that your error events have been fine-tuned to reflect real problems in your code, it's an excellent practice to react to errors as they happen. If an issue reflects a real problem in your code, resolve it; otherwise, delete and discard it.
```

Comment:
```suggestion
Now that your error events have been fine-tuned to reflect real problems in your code, it's an excellent practice to react to errors as they happen. If an issue reflects a real problem in your code, resolve it; otherwise, delete and discard it. Learn more in [Delete & Discard Issues](/product/accounts/quotas/#5-delete--discard-issues).
```